targets/xorg

```diff
@@ -43,7 +43,7 @@ inteldriver=''
 intelhasfbc=''
 pinmesa=''
 intelfbcsupport='y'
-if [ "${ARCH#arm}" = "$ARCH" ]; then
+if [ "${ARCH#arm}" = "$ARCH" ] && [ "${ARCH#aarch64}" = "$ARCH" ]; then
```

Comment:
Well in install/debian/defaults we set ARCH to arm64 while I did set it to aarch64. We should unify this. I tend to use aarch64, as arm64 is only a kernel thing, but I don't care too much honestly.